lib/rubocop/cop/rails/output_safety.rb

```diff
@@ -0,0 +1,24 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of output safety calls like html_safe and
+      # raw.
```

Comment:
Add here some examples of code the cop would consider offensive. See other cops for reference.